src/main/java/com/example/bookstoreapp/validator/FieldMatchValidator.java

```diff
@@ -0,0 +1,38 @@
+package com.example.bookstoreapp.validator;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.lang.reflect.Field;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
```

Comment:
These changes should be made in the scope of different pr, please prepare pr with only relevant changes